### DISCUSSION TITLE
E0B84447CD884CBB13E3266E4A865183AF6C5631 Public Signing Key?

### DISCUSSION THREAD

**Discussion by githububub** [jarun/nnn#1629] [General]
Where do we find the pubkey used to verify the auth of 4.8? Looks like the signing key pair changed and the pubkey has not been uploaded to the usual pgp keyservers...

**Comment by jarun** [comment]
Uploaded to some popular servers. Try now.

**Comment by githububub** [comment]
Done. Built. Thank you!
